.pre-commit-config.yaml

```diff
@@ -4,16 +4,16 @@ ci:
   skip: [poetry-lock, poetry-export]
 repos:
   - repo: https://github.com/psf/black
-    rev: 22.12.0
+    rev: 23.3.0
```

Comment:
Does these versions get updated automatically? Is that what we wanted? @greglucas

Reply:
Yes, they do get updated automatically for us because of line 3 above. `autoupdate_schedule: 'quarterly'`, so every 3 months or so is what it should do.

We could remove that if we don't want it, but I think it is good to keep updated if possible and this helps us not have to worry about it and it runs all the tests and creates a PR automatically. We don't have to merge these PRs if we disagree, or someone else can take over. Like here it is failing, so we should look at why before merging anything.

Reply:
That's nice. I wanted to make sure we are aware of that.